docs/graphql/core/api-reference/relay-graphql-api/query.rst

```diff
@@ -0,0 +1,252 @@
+.. meta::
+   :description: Hasura Relay GraphQL API queries and subscriptions API reference
+   :keywords: hasura, docs, GraphQL API, API reference, query, subscription, relay
+
+.. _relay_graphql_api_query:
+
+Relay API Reference - Query / Subscription
+====================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 3
+  :local:
+
+**query** / **subscription** syntax
+-----------------------------------
+
+.. code-block:: none
+
+    query|subscription [<op-name>] {
+      connection-object [([argument])]{
+        connection-object-fields
+      }
+    }
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key
+     - Required
+     - Schema
+     - Description
+   * - op-name
+     - false
+     - Value
+     - Name query/subscription for observability
+   * - connection-object
+     - true
+     - ConnectionObject_
+     - Name of the table connection
+   * - argument
+     - false
+     - Argument_
+     - One or more of filter criteria, instructions for sort order or pagination
+
+**Example: Relay Query**
+
+.. code-block:: graphql
+
+     query {
+       author_connection(first: 2){
+         pageInfo{
+           hasNextPage
+           endCursor
+         }
+         edges{
+           cursor
+           node{
+             id
+             name
+             username
+           }
+         }
+       }
+     }
+
+**Example: Relay Subscription**
+
+.. code-block:: graphql
+
+     subscription {
+       author_connection(first: 2){
+         pageInfo{
+           hasNextPage
+           endCursor
+         }
+         edges{
+           cursor
+           node{
+             id
+             name
+             username
+           }
+         }
+       }
+     }
+
+.. note::
+
+    For more examples and details of usage, please see :ref:`this <relay_queries>`.
```

Comment:
@marionschleifer Please add reference to sample relay queries here.

Reply:
@rakeshkky we decided to get this merged and add examples in a separate PR later.